nodejs/eks/babel.config.js

```diff
@@ -1,6 +1,3 @@
 module.exports = {
-    presets: [
-      ['@babel/preset-env', {targets: {node: 'current'}}],
-      '@babel/preset-typescript',
-    ],
-  };
\ No newline at end of file
+    presets: [["@babel/preset-env", { targets: { node: "current" } }], "@babel/preset-typescript"],
```

Comment:
Unrelated - I know some people prefer to split out formatting changes but I'm a huge fan of auto-formatting and don't mind it.

I would really love to see us adopt some auto-formatting git hooks.